geode-apis-compatible-with-redis/src/test/java/org/apache/geode/redis/internal/netty/CoderTest.java

```diff
@@ -176,4 +180,25 @@ public void stripTrailingZeroFromDouble_correctlyStripsTrailingZero(byte[] input
     };
   }
 
+  @Test
+  public void verify_appendAsciiDigitsToByteBuf_conversions() {
+    verify_appendAsciiDigitsToByteBuf(Long.MAX_VALUE);
+    verify_appendAsciiDigitsToByteBuf(Long.MIN_VALUE);
+    verify_appendAsciiDigitsToByteBuf(Integer.MAX_VALUE);
+    verify_appendAsciiDigitsToByteBuf(Integer.MIN_VALUE);
+    verify_appendAsciiDigitsToByteBuf(Short.MAX_VALUE);
+    verify_appendAsciiDigitsToByteBuf(Short.MIN_VALUE);
+    for (long i = Byte.MIN_VALUE; i <= Byte.MAX_VALUE; i++) {
+      verify_appendAsciiDigitsToByteBuf(i);
+    }
+  }
+
+  private void verify_appendAsciiDigitsToByteBuf(long value) {
+    String expected = Long.toString(value);
+    ByteBuf buf = ByteBufAllocator.DEFAULT.heapBuffer();
+
+    Coder.appendAsciiDigitsToByteBuf(value, buf);
+
+    assertThat(buf.toString(Charset.defaultCharset())).isEqualTo(expected);
```

Comment:
I think we might want to explicitly use UTF-8 as the charset here, as that's what Redis and the Coder class use.

Reply:
done